cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -682,6 +679,27 @@ protected Affinity getMergedAffinity() {
         return initContainers;
     }
 
+    @Override
+    protected List<Container> getContainers() {
+
+        List<Container> containers = new ArrayList<>();
+
+        Container container = new ContainerBuilder()
+                .withName(name)
+                .withImage(getImage())
+                .withEnv(getEnvVars())
+                .withVolumeMounts(getVolumeMounts())
+                .withPorts(getContainerPortList())
+                .withLivenessProbe(createExecProbe(healthCheckPath, healthCheckInitialDelay, healthCheckTimeout))
+                .withReadinessProbe(createExecProbe(healthCheckPath, healthCheckInitialDelay, healthCheckTimeout))
+                .withResources(resources())
+                .build();
+
+        containers.add(container);
```

Comment:
I'm fine without using the variable because it's not necessary. Regarding the singletonList, we are going to add a new container here in the coming days so we'll have to remove the singletonList in any case.

Reply:
You can replace it with `Arrays.asList()` :smile: